### PR TITLE
Complete Project

### PR DESCRIPTION
**Pull request by badger-99**
This PR proposes moving the completed **Space Traveler's Hub** project by @badger-99 and @majdalkilany to the `master` branch. The project includes the following features:
- Separate pages for Rockets, Missions. and the user's profile.
- The ability reserve a rocket or to cancel your reservation for for any of the rockets.
- The ability to join or leave a mission.
- A **My Profile** page that keeps track of missions you have joined. and rockets you have reserved.
- Dynamic elements that are rendered depending on the reservation status of a rocket or a mission.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
```
Comment (usorfaitheloho) [inline README.md]: You did well with the group capstone project, kindly note that you are expected to use the Microverse template for your README file so as to include all the details needed for your project. please use [this](https://github.com/microverseinc/readme-template) link 😄

File: README.old.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# space-travelers-hub
```
Comment (usorfaitheloho) [inline README.old.md:1]: Kindly delete this README file as it is not useful and shouldn't be in your project directory 👍

File: src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (usorfaitheloho) [inline src/setupTests.js:5]: Please note that as part of the requirement, you are expected to write Tests for your project 👍 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @badger-99  and @majdalkilany  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Used REACT REDUX ✔️
> * Descriptive pull request ✔️
> * Used REACT ROUTER✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @badger-99 and @majdalkilany, 

# STATUS: APPROVED 😃 

leonardo-dicaprio-leo-dicaprio

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights

- Use React components ✔️ 
- Connect React and Redux ✔️ 
- All tests passed ✔️ 

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**